Packs/FeedMISP/Integrations/FeedMISP/FeedMISP_test.py

```diff
@@ -448,3 +448,16 @@ def test_update_indicator_fields(
 ])
 def test_get_ip_type(indicator, indicator_type):
     assert get_ip_type(indicator) == indicator_type
+
+
+def test_search_query_indicators_pagination(mocker):
+    client = Client(base_url="example",
+                    authorization="auth",
+                    verify=False,
+                    proxy=False,
+                    timeout=60)
+    mocker.patch.object(Client, '_http_request', return_value={'response': {'Attribute': ['data1', 'data2']}})
```

Comment:
This unit test will have an infinite loop, you return the same response for each request and run in the method until an empty response is received.
Please fix this UT to suit the use case and fix the infinite loop.